.github/workflows/ci.yml

```diff
@@ -34,6 +34,10 @@ jobs:
           command: fmt
           args: -- --check
 
+      - name: temporary workaround - fmt all files under src
+        # Workaround for rust-lang/cargo#7732
+        run: cargo fmt -- --check $(find . -name '*.rs' -print)
```

Comment:
I think this should be able to just be what is put in the `args` section above, and we just have one format check.

```yml
      - name: cargo fmt -- --check
        uses: actions-rs/cargo@v1
        with:
          command: fmt
          args: -- --check $(find . -name '*.rs' -print)
```